donate/settings/environment.py

```diff
@@ -8,6 +8,10 @@
 # we rely on it being explicitly set (no default values) so that
 # we error out first.
 env = environ.Env(
+    ACOUSTIC_TX_CLIENT_ID=(str, ''),
```

Comment:
Added 4 acoustic env vars, plus one more called DONATION_RECEIPT_METHOD which is used to tell the app whether or not to send the receipt emails using the donate stack